### PR TITLE
OSDOCS-1649: Adding etcd encryption check for OAuth API server

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1649

Preview: https://osdocs-1649--ocpdocs.netlify.app/openshift-enterprise/latest/security/encrypting-etcd.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
WIP until this can be tested.

**Comment by bergerhoffer** [comment]
FYI @p0lyn0mial I tested this today and updated this PR, if you get a chance to look it over. Thanks!

Preview: https://osdocs-1649--ocpdocs.netlify.app/openshift-enterprise/latest/security/encrypting-etcd.html

File: modules/disabling-etcd-encryption.adoc [hunk 1]
```diff
@@ -70,3 +70,20 @@ Encryption mode set to identity and everything is decrypted
...
++
+The output shows `DecryptionCompleted` upon successful decryption:
++
+[source,terminal]
+----
+DecryptionCompleted
+Encryption mode set to identity and everything is decrypted
+----
++
+If the output shows `DecryptionInProgress`, this means that decryption is still in progress. Wait a few minutes and try again.
```
Comment (kalexand-rh) [inline modules/disabling-etcd-encryption.adoc:89]:
```suggestion
If the output shows `DecryptionInProgress`, decryption is still in progress. Wait a few minutes and try again.
```
If you take this suggestion, please apply it both modules.

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, but this looks good!

**Pull request merged**